internal/cmd/scraper/scraper.go

```diff
@@ -313,7 +313,7 @@ func Run(cfg *Config) error {
 			emitters = append(emitters, emitter)
 		case "infra-sdk":
 			s := synthesis.NewSynthesizer(cfg.EntityDefinitions)
-			emitter := integration.NewInfraSdkEmitter(s)
+			emitter := integration.NewInfraSdkEmitter(cfg.IntegrationMetadata, s)
```

Comment:
codeclimate is complaining about something here I do not understand

Reply:
agree thanks for seeing this. changed